Dockerfile

```diff
@@ -142,9 +142,9 @@ chmod a+x /build/software/java/jdk-6u33-linux-x64.bin \
 && rm /build/software/java/jdk-6u33-linux-x64.bin
 
 RUN \
-wget -P /build/software/nodejs https://nodejs.org/dist/v8.8.1/node-v8.8.1-linux-x64.tar.xz \
-&& tar -xvf /build/software/nodejs/node-v8.8.1-linux-x64.tar.xz --directory /build/software/nodejs \
-&& rm /build/software/nodejs/node-v8.8.1-linux-x64.tar.xz
+wget -P /build/software/nodejs https://nodejs.org/dist/v10.16.2/node-v10.16.2-linux-x64.tar.xz \
```

Comment:
Anyone else still depend on nodejs 8 for builds? 

cc @mohanvive @jeradrutnam @lalaji

Reply:
There are others jobs using nodejs 8. Hence, we can't remove the older version.

Reply:
@sajithshn  @CharukaK @lasanthaS Please check & validate

Reply:
@maheshika , I have updated the PR to include both versions(v8 & v10) of NodeJS. Hope we could switch to required NodeJS version from Jenkins configuration by specifying the respective paths?

Reply:
I have updated the Jenkins master. Will proceed with building the docker image and configure the respective build.